build.rs

```diff
@@ -9,36 +9,63 @@ use std::path::Path;
 use std::path::PathBuf;
 use std::process::Command;
 
-fn main() {
-    println!("cargo:rerun-if-changed=v8_c_api/src/v8_c_api.h");
-    println!("cargo:rerun-if-changed=v8_c_api/src/v8_c_api.cpp");
+const V8_VERSION: &str = "10.8.168.21";
```

Comment:
@vityafx this is the default version in case the `V8_VERSION` env var is not set, will change the name.
```
et version = env::var("V8_VERSION").unwrap_or(V8_VERSION.into());
```